tests/Tests/700__sandbox/700__sandbox.robot

```diff
@@ -0,0 +1,115 @@
+*** Settings ***
+Documentation    Test Suite for Sandbox Test cases
+Resource        ../../Resources/Common.robot
+Resource        ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
+Library         String
+Library         SeleniumLibrary
+Suite Setup     Begin Web Test
+Suite Teardown  End Web Test
+
+
+*** Variables ***
+${SANDBOX_AUTH_TYPE}   *******
```

Comment:
This variables probably should be in test-variables.yaml

Reply:
Removed